src/clj/puppetlabs/services/jruby_pool_manager/jruby_core.clj

```diff
@@ -162,6 +166,8 @@
   [config :- {schema/Keyword schema/Any}]
   (-> config
       (update-in [:compile-mode] #(keyword (or % default-jruby-compile-mode)))
+      (update-in [:profiling-mode] #(keyword (or % default-jruby-profiling-mode)))
+      (update-in [:profiler-output-file] #(or % nil))
```

Comment:
After talking this over with @lindboe I think I'm going to update the default here to a randomly named file that would be relative to the current working dir of the service, unless there are objections.

Reply:
With no file set the default behavior would be to dump profiling to stderr, which I think might go to different places on different platforms (probably the -daemon log on sysv platforms and journalctl on systemd platforms).

Reply:
+1 to the file in the working dir.